### PR TITLE
support multiple extra sources

### LINKED ISSUE #76: RFE: multiple extra src args
allow providing multiple extra src directories for layering content, rather than putting many sources in a single directory.

This will be either via making `${input_extra_src_dir}` a bash array, and appending to it via `getopts`
OR
Doing something nasty like comma or colon delimiting the `${OPTARG}` already there, so a single flag does the trick for handling multiple directories.

### PR DESCRIPTION
**Pull request by vrothberg**
Support specifying `-e $EXTRA_SOURCE_DIR` multiple times.
The EXTRA_SRC_DIR environment variable has been removed.

Fixes: #76
Signed-off-by: Valentin Rothberg <rothberg@redhat.com>

### CODE REVIEW

**Comment by vrothberg** [comment]
@lcarva @rhatdan @vbatts PTAL

File: Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM docker.io/library/fedora:31
+FROM docker.io/library/fedora:33
```
Comment (rhatdan) [inline Dockerfile]: Should we use the fedora from fedora rather then through Docker?

**Comment by rhatdan** [comment]
LGTM

File: Dockerfile (hunk 1 above)
Comment (vrothberg) [reply Dockerfile]: Good call!  Avoiding Docker Hub in CI is desirable with the latest rate limits.

**Review by lcarva — APPROVED** [review]

**Pull request merged**